packages/payload/src/graphql/schema/withOperators.ts

```diff
@@ -207,16 +207,12 @@ const defaults: DefaultsType = {
                 }
               }
 
-              if (typeof option === 'string') {
-                return {
-                  ...values,
-                  [option]: {
-                    value: option,
-                  },
-                }
+              return {
+                ...values,
+                [formatName(option)]: {
```

Comment:
We we'ren't formatting text options before?
Is this an unacaught bug or are we doing this somewhere else also?

Reply:
It was an uncaught bug, we were doing it correctly for radio above: https://github.com/payloadcms/payload/pull/3870/files/b918b99f6f385ff61ca6b4fe396fd345b89e5547#diff-a9767f97c784c2103e2aca5a105cab436d3c8dc156ea64df53b0ea9a3ae120f8L166-L170